install/top-level-packages.rst

```diff
@@ -30,32 +30,28 @@ In addition to the contents of ``lsst_apps``, it provides the following packages
 - `ap_verify <https://github.com/lsst/ap_verify>`_
 - `cbp <https://github.com/lsst/cbp>`_
 - `cp\_pipe <https://github.com/lsst/cp_pipe>`_
-- `ctrl\_execute <https://github.com/lsst/ctrl_execute>`_
```

Comment:
How are we really meant to be keeping this list up to date? I see I need to add display_astrowidgets, cp_pipe, drp_pipe, faro, sdm_schemas, and atmospec here. Also ctrl_mpexec and fgcm and eigen shouldn't be in the list either.

Reply:
It would seem possible to automate generation of the list, but I'm not sure where that automation would go or who would do it (likely requiring use of the eups API).